org.sabnzbd.SABnzbd.yaml

```diff
@@ -0,0 +1,112 @@
+app-id: org.sabnzbd.SABnzbd
```

Comment:
as I mentionned previously upstream as an all lower case app-id, and you should use that here.

This is not an option.

Reply:
Sorry, now I understand, fixed. :)